src/styles/dialog.css

```diff
@@ -3,8 +3,8 @@
  * @author Jamie jamie.h@inwinstack.com
  */
 
-.md-confirm-dialog-warn-theme md-dialog-actions button:last-child {
-  color: #FF6D00;
+.md-default-theme md-dialog-actions button:last-child {
```

Comment:
If I use the `theme()` method, the browser console will has some warning message when I clicked the button to open confirm dialog:
```
attempted to use unregistered theme 'confirm-dialog-warn'
Attempted to use unregistered theme 'confirm-dialog-warn'. Register it with $mdThemingProvider.theme().
Attempted to use unregistered theme 'confirm-dialog-warn'. Register it with $mdThemingProvider.theme().
attempted to use unregistered theme 'confirm-dialog-warn'
Attempted to use unregistered theme 'confirm-dialog-warn'. Register it with $mdThemingProvider.theme().
<md-backdrop> may not work properly in a scrolled, static-positioned parent container.
```
In the [codepen example](http://codepen.io/jamie3312/pen/dMzogN?editors=1010) also has the issue. Do you have any idea to resolve the issue?